lib/export-task.js

```diff
@@ -85,7 +109,8 @@ async function insertNewTask() {
        GRAPH <${process.env.MU_APPLICATION_GRAPH}> {
            ${sparqlEscapeUri(taskUri)} a export:Task ; 
                 mu:uuid ${sparqlEscapeString(taskId)} ;
-                export:status "ongoing" .
+                export:status "ongoing";
+                export:numberOfRetries 0.
```

Comment:
I would make the `export:numberOfRetries` optional on new tasks. That means, to select tasks to retry, select all tasks in the failed state without `export:numberOfRetries` or `export:numberOfRetries` < max number of attempts.